engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/discovery.controller.js

```diff
@@ -109,15 +109,18 @@ angular.module('Bastion.products').controller('DiscoveryController',
             baseUrl = $scope.discovery.url;
 
             toRet = _.map(urls, function (url) {
-                var params;
+                var params, urlPath, baseUrlPath;
 
                 params = {
                     url: $scope.discovery.url,
                     label: '',
-                    contentType: $scope.discovery.contentType
+                    contentType: $scope.discovery.contentType,
+                    repositoryUrl: url
                 };
                 if ($scope.discovery.contentType === 'yum') {
-                    params.path = url.replace(baseUrl, "");
+                    urlPath = new URL(url).pathname;
+                    baseUrlPath = new URL(baseUrl).pathname;
+                    params.path = urlPath.replace(baseUrlPath, "/");
```

Comment:
This could be because of the refactoring I suggested, I am seeing double slashes, e.g. `//brew3/`.

Changing to 

```javascript
                    params.path = urlPath.replace(baseUrlPath, "");
```

worked for me, though the custom autentication repos don't have a leading slash and the zoo repos do. I'm not sure why, but it doesn't seem like a huge issue, as long as there are no double slashes